code/src/index.css

```diff
@@ -11,3 +15,118 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.fa-grin-hearts {
+  margin-right: 10px;
+}
+
+.fa-envelope {
+  margin-left: 10px;  
+}
+
+h1 {
+  font-family: 'Pacifico', cursive;
+  text-align: center;
+  font-weight: 400;
+  color: #bfbfe8;
+}
+
+.message-container {
+  width: 90%;
+  max-width: 375px;
+  margin: 20px auto;
+  border: 1px #000 solid;
+  padding: 10px;
+  box-shadow: 5px 5px 0 #000;
+}
+
+/* MessageForm */
+
+.form {
+  background-color: #F2F0F0;
+}
+
+.form-label {
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.textarea {
+  margin: 7px 0;
+  resize: none;
+  overflow: hidden;
+  width: 100%;
+  font-size: 14px;
+  padding: 5px;
+  outline: none;
+}
+
+.submit-button {
+  background-color: rgb(255,173,173);
+  display: block;
+  border-radius: 15px;
+  border: none;
+  padding: 8px 15px;
+  font-size: 12px;
+  outline: none;
+  cursor: pointer;
+}
+
+.submit-button:hover {
+  background-color: rgb(250, 141, 141);
+}
+
+.submit-button:active {
+    transform: translateY(2px);
+}
+
+/* MessageElement */
+
+.message {
+  margin: 3px 0 13px 0;
+  font-weight: 500;
+  overflow-wrap: break-word;
+}
+
+.flex-wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
+.likes-button {
+  border-radius: 50%;
+  border: none;
+  padding: 12px;
+  outline: none;
+  cursor: pointer;
+}
```

Comment:
It looks like the circle surrounding the heart icon is not really a circle, more oval.  A suggestion is to put a fixed width and height the surrounded "round" element .likes-button{width: 30px; height: 30px;} I think then the element will be more like a circle. (Just a suggestion, nothing you need to change, if you don't want to☺️